pipelines/controller/src/main/java/com/google/fhir/analytics/DwhFilesManager.java

```diff
@@ -425,31 +425,27 @@ List<String> findExistingViews(String viewRoot, String viewDefinitionsDir)
 
   private int getLastIndexOfSlash(String dwhRootPrefix) {
     CloudPath cloudPath = DwhFiles.parsePath(dwhRootPrefix);
-    int index = -1;
-    switch (cloudPath.getScheme()) {
-      case LOCAL_SCHEME:
-        index = dwhRootPrefix.lastIndexOf(File.separator);
-        break;
-      case GCS_SCHEME:
-      case S3_SCHEME:
+    return switch (cloudPath.getScheme()) {
+      case LOCAL_SCHEME -> dwhRootPrefix.lastIndexOf(File.separator);
+      case GCS_SCHEME, S3_SCHEME -> {
         // Fetch the last index position of the character '/' after the bucket name in the gcs path.
         String gcsObject = cloudPath.getObject();
         if (Strings.isNullOrEmpty(gcsObject)) {
-          break;
+          yield -1; // The original `break` provided no value to return.
```

Comment:
nit: I don't think the comment is necessary; I think the intention was to return `-1` before by returning the `index` (as you are doing now).

Reply:
Done